fiware-rss/src/main/java/es/upm/fiware/rss/model/RSSReport.java

```diff
@@ -105,5 +117,7 @@ public void setCurrency(String currency) {
         this.currency = currency;
     }
 
-    
+    public boolean getPaid() { return paid; }
+
+    public void setPaid(boolean paid) { this.paid = paid; }
```

Comment:
If paid is a boolean value, by convention the get method should be called 'isPaid'